lessons/frameworks.ipynb

```diff
@@ -0,0 +1,1261 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "# Statistical Frameworks\n",
+    "\n",
+    "Goals:\n",
+    "\n",
+    "* Explore the relationship between \"characterizing the posterior PDF\" and \"fitting a model to data.\"\n",
+    "\n",
+    "* Understand how to derive maximum likelihood estimators and their confidence intervals\n",
+    "\n",
+    "* Be able to compare, contrast and appreciate the Bayesian and Frequentist approaches to statistics\n",
+    "\n",
+    "We'll work through a simple Bayesian inference (fitting a straight line and reporting summaries of marginalized posterior PDFs), and then compare with the frequentist procedure of _estimating_ the gradient and intercept and reporting confidence intervals."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Further reading\n",
+    "\n",
+    "* Ivezic 3.2.2, 4.1 and 4.2\n",
+    "\n",
+    "* [Vanderplas (2014), \"Frequentism and Bayesianism: A Python-driven Primer\"](https://arxiv.org/abs/1411.5018)\n",
+    "\n",
+    "* [Hogg, Bovy & Lang (2010), \"Data analysis recipes: Fitting a model to data\"](https://arxiv.org/abs/1008.4686)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## The period-magnitude relation in Cepheid stars\n",
+    "\n",
+    "* Cepheids are stars whose brightness oscillates with a stable period the logarithm of which appears to be strongly correlated with their mean luminosity (or absolute magnitude).\n",
+    "\n",
+    "<img width=\"512\" src=\"graphics/Cepheid-variabledb92-600x461.jpg\"/>"
```

Comment:
Need to check that we have the right to distribute this graphic, and include a .source file for it. I had been attempting to make each static graphic hyperlink to its .source, but haven't been able to keep up with it.

Reply:
I found the source file, its checked in.